mixer/pkg/server/server.go

```diff
@@ -211,11 +216,17 @@ func newServer(a *Args, p *patchTable) (server *Server, err error) {
 
 	var namespaces []string
 
-	if a.WatchedNamespaces != "" {
-		namespaces = strings.Split(a.WatchedNamespaces, ",")
-		namespaces = append(namespaces, a.ConfigDefaultNamespace)
+	if a.MemberRollName != "" {
+		mrc, err := meshcontroller.NewMemberRollControllerFromConfigFile(a.ConfigStoreURL, a.MemberRollNamespace, a.MemberRollName, a.MemberRollResync)
+		if err != nil {
+			_ = s.Close()
+			return nil, multierror.Prefix(err, "Could not create MemberRollController.")
+		}
+		s.mrcStop = make(chan struct{})
+		s.mrc = mrc
+		mrc.Start(s.mrcStop)
 	} else {
-		namespaces = []string{metav1.NamespaceAll}
```

Comment:
isn't this the same?

Reply:
Yeah, but the commit this is based on uses `""` for whatever reason, so I just kept this whole block as-is. I hope this gone the next time we do this anyway.